src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -17,17 +17,12 @@ public static class ApplicationInsightsExtensions
     {
         public static IApplicationBuilder UseApplicationInsightsRequestTelemetry(this IApplicationBuilder app)
```

Comment:
OK. The question is closed. Thanks for sharing your thoughts on this, @Eilon & @SergKanz.